frontEnd/src/main/java/olumofe/raphael/frontEnd/CarController.java

```diff
@@ -0,0 +1,35 @@
+package olumofe.raphael.frontEnd;
+
+import org.springframework.web.bind.annotation.*;
+import starterProject.domain.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicLong;
+@org.springframework.stereotype.Controller
+
+public class CarController {
+
+    Transmission automatic = Transmission.builder().automatic(true).build();
+    Wheels parelli = Wheels.builder().wheels(4).build();
+    Engine v12 = Engine.builder().cylinders(12).build();
+    Doors verticalDoors = Doors.builder().doorNumbers(2).build();
+    @PostMapping("/createCarList")
+    @ResponseBody
+    public List<Car> createCarList(){
+        Car ferarri = Car.builder().transmission(automatic).wheels(parelli).engine(v12).doors(verticalDoors).name("ferrari").build();
+        Car buggatti = Car.builder().transmission(automatic).wheels(parelli).engine(v12).doors(verticalDoors).name("bugatti").build();
+        List <Car> objectList =  new ArrayList<Car>();
+        objectList.add(ferarri);
+        objectList.add(buggatti);
+        return objectList;
+    }
+    @PostMapping("/car")
+    @ResponseBody
+    public Car car(){
```

Comment:
This should take a Car object in the method

```
public Car car(Car car){
}
```
It should then store it in a data structure